src/olmo_core/data/numpy_dataset.py

```diff
@@ -235,7 +242,7 @@ def _warmup_clients(self):
 
     def map(
         self,
-        func: Callable[[PathOrStr], T],
+        func: Callable[[Tuple[PathOrStr, int]], T],
```

Comment:
Can you update the docstring of this method to reflect the new signature? Also, why not `func: Callable[[PathOrStr, int], T]` instead of `Callable[[Tuple[PathOrStr, int], T]`?

Reply:
No reason, personal preference :) I switched it back to a list of positional args.